proj/src/main/webapp/README.md

```diff
@@ -0,0 +1,12 @@
+$ npm install browserify -g
+$ cd step14-2020/proj/src/main/webapp
+$ npm install
+$ cd ../../..
+$ browserify src/main/webapp/app.js -o src/main/webapp/bundle.js
+$ mvn package appengine:run
+
```

Comment:
Can you also add a bit on if you want to deploy.
If you want to deploy,  first you need to delete the `step14-2020/proj/target` and `` directories and then you can call `$ browserify src/main/webapp/app.js -o src/main/webapp/bundle.js` then `$ mvn package appengine:deploy`. Once you are done deploying, you need to go back to `step14-2020/proj/src/main/webapp` and run `$ npm install`.

It's a confusing process that I don't think people will pick up on their own. Normally we shouldn't be deploying from our local copy, but just in case we need to for any reason, it's good to have it documented how to do it.

Reply:
Deploying can also be done using the clouddeploy.json config file, so if you want, you can try writing instructions that run that using the local cloud build steps.
https://cloud.google.com/cloud-build/docs/build-debug-locally